test/fixtures/output/r/httr/headers.r

```diff
@@ -0,0 +1,7 @@
+library(httr)
+
+url <- "http://mockbin.com/har"
+
+response <- VERB("GET", url, add_headers(x_foo = 'Bar'), content_type("application/octet-stream"), accept(application/json))
```

Comment:
I'm not familiar with R, but should the `application/json` have quotes around it?

Reply:
Yes! You're right, I have corrected it now. Thank you!